apps/files/src/components/FileList.vue

```diff
@@ -60,7 +60,7 @@
             >
               <div v-if="checkboxEnabled">
                 <oc-checkbox
-                  class="uk-margin-small-left"
+                  class="uk-margin-small-left files-list-checkbox"
```

Comment:
You are already using the size property (needs to be changed to `large` though). Don't use the `files-list-checkbox` class (it's removed already anyway).